qa/run-standalone.sh

```diff
@@ -20,7 +20,8 @@ function get_python_path() {
             py_ver=3
         fi
     fi
-    echo $(realpath ../src/pybind):$(pwd)/lib/cython_modules/lib.$py_ver
+    python_common=$(realpath ../src/python-common)
+    echo $(realpath ../src/pybind):$(pwd)/lib/cython_modules/lib.$py_ver:$python_common
```

Comment:
We have a similar thing in

https://github.com/ceph/ceph/blob/3a781f9577dac407f0f57275dc8df5a0141ac5c5/src/pybind/mgr/dashboard/run-backend-api-tests.sh#L125

I wonder if it would make sense to create a single virtual python environment containing all our Python modules (including the mgr modules) instead of fiddling with PYTHONPATH in half a dozen locations.